librz/core/panels.c

```diff
@@ -131,7 +131,7 @@ static char *menus_Debug[] = {
 };
 
 static const char *menus_Analyze[] = {
-	"Function", "Symbols", "Program", "BasicBlocks", "Calls", "References",
+	"Function", "Symbols", "Program", "Calls", "References",
```

Comment:
Any particular reason why you removed this from panels mode?

Reply:
Because it was executing `rz_cmd_analysis_blocks`, which is essentially the `aab` command, that I removed as well.